src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/RegularExpressionAttribute.cs

```diff
@@ -31,6 +31,11 @@ public RegularExpressionAttribute([StringSyntax(StringSyntaxAttribute.Regex)] st
         /// </summary>
         public int MatchTimeoutInMilliseconds { get; set; }
 
+        /// <summary>
+        /// Gets or sets the timeout to use when matching the regular expression pattern
```

Comment:
> Gets or sets

I don't see a setter.  Seems like we should have one as it wouldn't make sense to only support getting as `TimeSpan`

Reply:
```diff
!       This one might not end up being that useful, since people generally don't ever manipulate an instance of this attribute.
+       public TimeSpan MatchTimeout { get; }
```
I think that is so wanted that there is no setter (I have to adapt the documentation of course)

@terrajobst Can you verify that?

Reply:
I have adjusted this despite the fact that we are waiting for the answer from Immo.

Reply:
@terrajobst What do you think?

Reply:
@bartonjs with your API review hat on should we add a setter (was not in approved shape) for symmetry with the int property?

Reply:
If there's contention then the best answer is probably to bring it up as a separate item to API Review, so we can discuss it in the larger group (if we want to keep it in this PR, then a blocking issue so it floats to the top).

My initial thought was "no", remembering the guideline as "avoid inter-dependent properties"; but it turns out that's just my rephrasing of one nuance of the actual guideline:

> DO allow properties to be set in any order, even if this results in a temporary invalid state of the object.

With the Guideline As Written, it wouldn't be bad, just that the last writer wins.  So, it's probably goodness to have... but since the original proposal didn't have it, and the review discussion didn't add it... it's probably worth having a chat again.

Reply:
@deeprobin Let's leave it out for now to get this PR done

Reply:
Reading again.... this is an attribute.  The 99.99% use case for setting properties on attributes is via `[UsingAnAttribute(..., Property = Value)]`.  Since there is no attribute-literal syntax for a TimeSpan, the setter doesn't have much value.

Get-only seems more appropriate to me in this context.